alpenhorn/client.py

```diff
@@ -692,24 +692,24 @@ def clean(node_name, days, size, force, now, target, acq):
     type=str,
     default=None,
 )
-def mounted(host):
-    """list the nodes mounted on this, or another specified, machine"""
+def active(host):
```

Comment:
You should also rename the `mount` command below.

Also, this is a CLI change.  Do we want to have legacy aliases `mounted`/`mount`?

Reply:
This `mount` command:
```
@cli.command()
@click.argument("name")
@click.option("--path", help="Root path for this node", type=str, default=None)
@click.option("--user", help="username to access this node.", type=str, default=None)
@click.option(
    "--address", help="address for remote access to this node.", type=str, default=None
)
@click.option(
    "--hostname",
    help="hostname running the alpenhornd instance for this node (set to this hostname by default).",
    type=str,                                                                                            
    default=None,
)
def mount(name, path, user, address, hostname):
    """Interactive routine for mounting a storage node located at ROOT."""

    # We need to write to the database.
    db.connect(read_write=True)

    try:
        node = di.StorageNode.get(name=name)
    except pw.DoesNotExist:
        print('Storage node "%s" does not exist. I quit.' % name)

    if node.active:
        print('Node "%s" is already active.' % name)
        return
```

Reply:
Yeah sure, I can add `mount` and `mounted` commands that call `active`.

Reply:
Yeah that mount command.  It should be renamed to `activate`.  I realise it _does_ actually try to mount a volume if necessary but it's real job is to activate the node in the DB.  (Then point `mount` at `activate`, not `active`).

Reply:
Now that I think about it, there's probably an `unmount` command as well that should get changed to `deactivate`.

Reply:
@ketiltrout I've renamed `unmount` -> `deactivate` and kept an alias. I also noticed `unmount/mount_transport`, these functions actually call `mount/umount` commands so I decided to leave them as is.